src/utils.rs

```diff
@@ -0,0 +1,6 @@
+const INVALID_CHARACTERS: [char; 5] = ['\u{202A}', '\u{202B}', '\u{202C}', '\u{202D}', '\u{202E}'];
+/// This method strips all occurances of the RTLO Unicode character.
+/// [Why is this needed](https://github.com/deltachat/deltachat-core-rust/issues/3479)?
+pub(crate) fn create_safe_string(input_str: &str) -> String {
```

Comment:
"safe string" is a bit broad :) maybe name the function closer to what it is actually doing, sth. as  `strip_rtlo_characters()`. this makes the comment mostly superfluous.

accordingly, the array would be named just `RTLO_CHARACTERS`.

also, we already have `tools.rs` - so better move the function there. i doubt, there is a sharp, obvious distinction between "tools" and "utilities" and having both would end in some mess and random placing of helper functions.

Reply:
unresolving, what about the naming?